src/UI/Component/Input/Field/Factory.php

```diff
@@ -205,4 +205,46 @@ public function dependantGroup(array $inputs);
 	 * @return    \ILIAS\UI\Component\Input\Field\Checkbox
 	 */
 	public function checkbox($label, $byline = null);
+
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     A textarea is intended for entering multi-line texts.
+	 *   composition: >
+	 *      Textarea fields will render an textarea HTML tag.
+	 *   effect: >
+	 *      Textarea inputs are NOT restricted to one line of text.
+	 *   rivals:
+	 *      text field: Use a text field if users should input only one line of text.
+	 *      numeric field: Use a numeric field if users should input numbers.
+	 *      alphabet field: >
+	 *          Use an alphabet field if the user should input single letters.
+	 *
+	 * rules:
+	 *   usage:
+	 *     1: Textarea Input MUST NOT be used for choosing from predetermined options.
+	 *     2: >
+	 *         Textarea input MUST NOT be used for numeric input, a Numeric Field is
+	 *         to be used instead.
+	 *     3: >
+	 *         Textarea Input MUST NOT be used for letter-only input, an Alphabet Field
+	 *         is to be used instead.
+	 *     4: >
+	 *         Textare Input MUST NOT be used for single-line input, a Text Field
+	 *         is to be used instead.
+	 *   interaction:
+	 *     1: >
+	 *         Textarea Input MAY limit the number of characters, if a certain length
+	 *         of text-input may not be exceeded (e.g. due to database-limitations).
```

Comment:
Jour Fixe, 23 APR 2018 : Please add to rules: "If a min. or max. number of characters is set for textarea, a byline MUST be added stating the number of min. and/or max. characters."